### DISCUSSION TITLE
Version 3.2.23

### DISCUSSION THREAD

**Discussion by rioujy** [eclipse-birt/birt#1647] [Q&A]
Hello I am a beginner in BIRT, I need version 3.2.23 of Birt, my ERP does not accept reports modified with higher versions. How do I do it and where to download? Thanks in advance

**Comment by merks** [comment]
I don't think such an ancient version exists anymore.

https://archive.eclipse.org/birt/

**Reply by speckyspooky** [reply]
An addition path is this which is not directly linked but I cannot see your historical version or BIRT.
https://archive.eclipse.org/birt/downloads/drops/

My by you can use an older designer based on 3.7 witch is available under my noted link
and you can try is the version number correct. If not it would be possible to change the version number at the xml-file directly (with an text editor) to the version number which your ERP-BIRT needs.
